docs/source/usage/benchmarks.rst

```diff
@@ -8,15 +8,14 @@ Parallel benchmark 8
 
 Build based on the helper functions in the :ref:`benchmark utilities <utilities-benchmark>`, this benchmark executes a simple parallel read-write test.
 
-In particular, this test case writes and reads a 4D array of type ``uint64_t``, sliced 1D along the first dimension.
+In particular, this test case writes and reads a 3D array of type ``uint64_t``, sliced 1D along the first dimension.
```

Comment:
@franzpoeschel looking at this again, writing `double`s would probably be more realistic :sweat_smile: